store/store.go

```diff
@@ -266,8 +293,15 @@ func (s *Store) HasByHeight(ctx context.Context, height uint64) (bool, error) {
 }
 
 func (s *Store) hasByHeight(height uint64) (bool, error) {
-	_, err := s.cache.Get(height)
```

Comment:
This was a bug because we didn't close the Accessor after Get. Warrants adding Has method to the cache.